Dockerfile

```
@@ -45,7 +46,8 @@ RUN apt-get update && \
 WORKDIR /usr/src/app
 
 # Keep the .git directory in order to properly report version
-COPY . /usr/src/app
+COPY . ./
```

Only thing that needs to be copied is package.json if we intend to use yarn to launch things
```suggestion
COPY ./package.json ./
```